qmd/protocols.qmd

```diff
@@ -0,0 +1,162 @@
+
+This flexible structure allows protocol steps and subprotocols to be reused in any number of protocols. The relationships between protocols, protocol orderings, and protocol steps are shown in Figure A.
+
+![Figure A: Main entities and relationships used to represent protocols](./assets/protocols/protocols.png)
+
+## Suggested minimum Reporting
+
+As mentioned above, we know that protocols can be very complicated, and taking the time to record every detail of a protocol can be time consuming, and potentially tedious. While having such a granular level of data can be rewarding in its own right, establishing minimum standards for what should be reported in a protocol can free up valuable time. Building on the work of (McClary-Gutierrez, et al. 2021)[https://pubs.rsc.org/en/content/articlehtml/2021/ew/d1ew00235j], we recommend that the minimum protocol information follow what is laid out in the table below (adapted from McClary-Gutierrez, et al., 2021).
+
+| Method or Measure to Record in Protocols           | Example Values                                      |
+|----------------------------------------------------|-----------------------------------------------------|
+| Pre-concentration storage temperature	             | Degrees Celsius, Degree Farenheit, etc.             |
+| Concentration method & citation	                   | PEG precipitation, ultrafiltration, etc.            |
+| Recovery control name & efficiency	               | BCoV, BRSV, MHV, none, etc.                         |
+| Extraction method & citation	                     | promega manual tna kit, qiagen powerwater kit, etc. |
+| Amount of sample processed	                       | Starting volume [mL] or mass [g] of raw sample      |
+| Extraction blanks results	                         | Signal not detected, blanks not used, etc.          |
+| Target quantification	PCR type	                   | qPCR, ddPCR, etc.                                   |
+| Target gene(s)	                                   | Gene target and primers/assay                       |
+| Endogenous wastewater control name & concentration | PMMoV, crAssphage, none, etc.                       |
+
+## Example Protocol: Methods Text to ODM Data - Sample Fractioning
+
+To show how protocol data can be reported in practice, we will use a portion of the methods section from (a paper from the Delatolla Research Group)[https://www.sciencedirect.com/science/article/pii/S0043135420310952?via%3Dihub] at the University of Ottawa. This group is among our closest collaborators, and have supported and worked substantially on the ODM.
+
+Lets start by looking at a chunk from the methods section of the paper. This section (Figure B) specifically is talking about different approaches of fractionating wastewater that were tested out while developing their protocol. 
+
+![Figure B: Methods Section Breakdown - Fraction](./assets/protocols/Methods-to-methods.png)
```

Comment:
The ellipses make it hard to understand what is being grouped as separate methods. Could you instead use differently-colored highlighting?